### ISSUE TITLE
Debian Release Mapping needs update

### ISSUE THREAD

**Issue by jzelinskie** [quay/clair#417]
https://github.com/coreos/clair/blob/fec86b621187501444a681908eeaa385ae68c18f/database/namespace_mapping.go#L26

Debian just moved stretch to stable, so we need to update this list.

**Comment by cmur2** [comment]
Also, there is now a new warning so a mapping for "buster" is also needed:

```
clair: fetcher note: Debian buster is not mapped to any version number (eg. Jessie->8). Please update me.
```

**Comment by jzelinskie** [comment]
This was fixed in #416. I'm working on cutting a v2.0.1 release with these changes!
